Resources/Views/Submissions/Fields/input-field.leaf

```diff
@@ -3,7 +3,7 @@
         <label for="#(key)">#(label)</label>
     }
     
-    #bs:input("#get(type)", "form-control #if(hasErrors) { is-invalid }", "id='#(key)' name='#(key)' #if(value) { value='#(value)' } autofocus #if(isRequired) { required } #if(placeholder) { placeholder='#(placeholder)' }")
+    #bs:input("#get(type)", "form-control #if(hasErrors) { is-invalid }", "id='#(key)' name='#(key)' #if(value) { value='#(value)' } #if(isRequired) { required } #if(placeholder) { placeholder='#(placeholder)' }")
```

Comment:
Good thing you removed autofocus as well. Why is `#get(type)` necessary? Can it be `#(type)`?

Reply:
Any thoughts on whether and how autofocus should be configurable?

Reply:
`#get()` isn't needed - removed that part. We could consider having an "additional attributes" parameter that could be based on to `bs:input`. Created an issue on it: https://github.com/nodes-vapor/submissions/issues/26